frontend/src/pages/team/changeType.vue

```diff
@@ -303,15 +307,43 @@ export default {
             }
         },
         sendContact: async function () {
-            if (this.input.teamType) {
-                billingApi.sendTeamTypeContact(this.user, this.input.teamType, 'Team: ' + this.team.name).then(() => {
-                    this.$router.push({ name: 'Team', params: { team_slug: this.team.slug } })
-                    Alerts.emit('Thanks for getting in touch. We will contact you soon regarding your request.', 'info', 15000)
-                }).catch(err => {
-                    Alerts.emit('Something went wrong with the request. Please try again or contact support for help.', 'info', 15000)
-                    console.error('Failed to submit hubspot form: ', err)
-                })
-            }
+            const user = this.user
+            Dialog.show({
+                header: 'Talk to sales to Upgrade to Enterprise',
+                kind: 'primary',
+                boxClass: 'ff-dialog-box--wide',
+                confirmLabel: 'Close',
+                canBeCanceled: false,
+                is: {
+                    // eslint-disable-next-line vue/one-component-per-file
+                    component: defineComponent({
+                        components: { ObjectProperties },
+                        computed: {
+                            url () {
+                                const url = new URL('flowfuse/book-a-demo-call', 'https://meetings-eu1.hubspot.com')
```

Comment:
This is not ideal, but for the time being it works. My reasoning is that for self hosted installations billing would have to be enabled and also the 'contact sales' check must be checked on a team type which shouldn't really happen

Ideally we should add a new parameter in the yaml config and expose that to the frontend